### PR TITLE
Updating default value of eps in RMSNorm documentation

### LINKED ISSUE #155527: Clarify default value of eps in RMSNorm documentation
### 📚 The doc issue

Hi PyTorch team,

I noticed an inconsistency between the documentation and the actual behavior of torch.nn.RMSNorm.

In the docs, the eps parameter is defined as:

eps (Optional[float]) – a value added to the denominator for numerical stability. Default: torch.finfo(x.dtype).eps

However, in the function signature, the default value is shown as None, which may be misleading. In reality, when eps=None is passed, it gets internally set to torch.finfo(x.dtype).eps.

It would be helpful for users if:
	1.	The function signature explicitly indicated that eps=None defaults to torch.finfo(x.dtype).eps.
	2.	The documentation clarified that this is dynamically determined at runtime based on the input dtype.

Thanks for your great work on PyTorch!

### Suggest a potential alternative/fix

_No response_

### PR DESCRIPTION
**Pull request by RajeshvShiyal**
In parameters description section of torch.nn.RMSNorm, we can change description for parameter eps is as below.

Existing eps parameter description:
eps (Optional[float]) – a value added to the denominator for numerical stability. Default: torch.finfo(x.dtype).eps()

Expected eps parameter description:
eps (Optional[float]) – a value added to the denominator for numerical stability. eps=None defaults to torch.finfo(x.dtype).eps(). This is dynamically determined at runtime based on the input dtype

Fixes #155527

### CODE REVIEW

**Review by spzala — COMMENTED** [review]
@RajeshvShiyal thanks! A quick feedback - in your PR description, please change "Fixes #ISSUE_NUMBER" to Fixes #155527
Also, make sure build failures related to CLA check and label are taken care. Thanks!

**Comment by RajeshvShiyal** [comment]
Hello @spzala, required changes have been done as per your suggestion.

**Comment by spzala** [comment]
@RajeshvShiyal please fix lint error - line too long (206 > 120 characters) https://github.com/pytorch/pytorch/actions/runs/15956398038/job/45050249796?pr=157223#step:15:182   (on line 346). Once you make changes, make sure you follow https://github.com/pytorch/pytorch/wiki/lintrunner to run lint locally successfully. Thanks!

**Comment by RajeshvShiyal** [comment]
Hello @spzala, 
Thank you for your review comment related to lint error. I will check it locally and will correct it.

**Comment by vadimkantorov** [comment]
Related:
- https://github.com/pytorch/pytorch/issues/41527#issuecomment-664552040
- https://github.com/pytorch/pytorch/issues/151110#issuecomment-2803117883

**Comment by RajeshvShiyal** [comment]
Hello @spzala,

After running lint locally, I was able to identify the lint error and same has been corrected. Thank you.

**Comment by RajeshvShiyal** [comment]
Hello @albanD, @mikaylagawarecki, @jbschlosser Can you please review the pull request.

CC: @spzala

**Comment by vadimkantorov** [comment]
@AaronWang04 in the new https://github.com/pytorch/pytorch/pull/153666#issuecomment-3019743039, is normalization done in fp32 always (regardless of the input dtype)? if so - unspecified `eps` should also probably be taken as `torch.finfo(torch.float32).eps().`?

Also, I didn't understand if eltwise scaling in the new fused impl is also always done in fp32 or not... It would be great to clarify these casts/compute dtypes in the docs, as these small details seem to matter

**Comment by AaronWang04** [comment]
> @AaronWang04 in the new [#153666 (comment)](https://github.com/pytorch/pytorch/pull/153666#issuecomment-3019743039), is normalization done in fp32 always (regardless of the input dtype)? if so - unspecified `eps` should also probably be taken as `torch.finfo(torch.float32).eps().`?
> 
> Also, I didn't understand if eltwise scaling in the new fused impl is also always done in fp32 or not... It would be great to clarify these casts/compute dtypes in the docs, as these small details seem to matter

normalization is done with ACC_TYPE which is double if the input is double but otherwise floats
https://github.com/pytorch/pytorch/blob/d36afac83b01c3de214db91f3d4b3f447f9a77b7/aten/src/ATen/AccumulateType.h#L130-L137

edit: btw the casting is done here 
https://github.com/pytorch/pytorch/blob/b4abf414254ed4d8779bad291dd0141097f019e7/aten/src/ATen/native/cuda/layer_norm_kernel.cu#L1154

**Comment by vadimkantorov** [comment]
If it's float32 or float64, should unspecified eps then also default to be `torch.finfo(torch.float32).eps` / `torch.finfo(torch.float64).eps`?

Is eltwise multiplication done in then in the original dtype (such as float16)? or also in float32/float64?

**Comment by AaronWang04** [comment]
> If it's float32 or float64, should unspecified eps then also default to be `torch.finfo(torch.float32).eps` / `torch.finfo(torch.float64).eps`?
> 
> Is eltwise multiplication done in then in the original dtype (such as float16)? or also in float32/float64?

float is float32, double is float64
every intermediate computation is done in the ACC_TYPE including elementwise operations
